at_common_flutter/lib/widgets/custom_toast.dart

```diff
@@ -0,0 +1,49 @@
+
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+
+class CustomToast {
+  /// pass [isError] true to have a red bg, [isSuccess] true to have a green bg.
+  static void show(String message, BuildContext context,
+          {Color? bgColor,
+          Color? textColor,
+          double? toastWidth,
+          double fontSize = 12,
+          int duration = 3,
+          double gravity = 10.0,
+          bool isError = false,
+          bool isSuccess = false}) {
+            assert(!(isError && isSuccess), 'Both isError and isSuccess cannot be true');
```

Comment:
@yahu1031 I think we can have a `dismissDirection` property as well.

Reply:
will add it.